nixos/modules/programs/qdmr.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
this pattern is discouraged and we're trying to get rid of it, see e.g. #209325

Reply:
Is [this](https://github.com/NixOS/nixpkgs/compare/7971374641bd1a2ab2848b17263cbfa719b99613..c049bb77a227ea7593640e36f557a1fee318eec5) better now?

Reply:
Either version is fine, IMO. Just don't do `with lib;`.